python/tank/platform/environment.py

```diff
@@ -329,8 +328,24 @@ def get_app_descriptor(self, engine_name, app_name):
 
         return d
 
-    def set_context(self, context):
```

Comment:
This was a method that I had previously added. Nothing is using it by the old set_context name.